pyrsa/vis/mds_plot.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
```

Comment:
We don't have automatic doc generation anymore, so a new module means a new rst file is required in the narrative docs.